#! /bin/sh /usr/share/dpatch/dpatch-run
## 38_script_manpages.dpatch by  <blade@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Varios small additional manpages

@DPATCH@
diff -urNad cdrtools-2.01.01~/doc/list_audio_tracks.man cdrtools-2.01.01/doc/list_audio_tracks.man
--- cdrtools-2.01.01~/doc/list_audio_tracks.man	1970-01-01 01:00:00.000000000 +0100
+++ cdrtools-2.01.01/doc/list_audio_tracks.man	2006-03-20 23:07:36.912212328 +0100
@@ -0,0 +1,29 @@
+.TH "LIST_AUDIO_TRACKS" "1" "Tue Feb 15 13:03:51 MST 2005" "" "list_audio_tracks"
+
+.PP
+.SH "NAME"
+list_audio_tracks is an alias for cdda2wav.
+.PP
+.SH "SYNOPSIS"
+.PP
+\fBlist_audio-tracks\fP
+.PP
+.SH "DESCRIPTION"
+.PP
+\fBlist_audio_tracks\fP equals to the invocation of cdda2wav, so all the
+arguments for the latter can be used.
+.PP
+.SH SEE ALSO
+cdda2wav(1)
+.PP 
+.SH "AUTHOR" 
+.PP 
+\fBcdda2was\fP was written by Joerg Schilling <js@cs\&.tu-berlin\&.de> and
+others.
+.PP 
+This manual page was written by Oleksandr Moskalenko
+<malex@tagancha\&.org>, for
+the Debian GNU/Linux system\&.  It may be used by other distributions
+without contacting the author\&.  Any mistakes or omissions in the
+manual page are my fault; inquiries about or corrections to this
+manual page should be directed to me (and not to the primary author)\&.
diff -urNad cdrtools-2.01.01~/doc/pitchplay.man cdrtools-2.01.01/doc/pitchplay.man
--- cdrtools-2.01.01~/doc/pitchplay.man	1970-01-01 01:00:00.000000000 +0100
+++ cdrtools-2.01.01/doc/pitchplay.man	2006-03-20 23:08:05.119924104 +0100
@@ -0,0 +1,37 @@
+.TH "PITCHPLAY" "1" "Tue Feb 15 12:53:23 MST 2005" "" "pitchplay"
+
+.PP
+.SH "NAME"
+pitchplay \- wrapper script to play audio tracks with cdda2wav with different
+pitches through a soundcard (cdda2wav must have soundcard support enabled).
+.PP
+.SH "SYNOPSIS"
+.PP
+\fBpitchplay\fP <track a> <pitch a> <track b> <pitch b>...
+.PP
+.SH "DESCRIPTION"
+.PP
+\fBpitchplay\fP allows playback of audio tracks with cdda2wav with pitches
+specified in percentage with 100% being the original pitch, 50% being one
+octave lower, 200% one octave higher.
+.PP
+.SH "EXAMPLES"
+.PP
+\fBpitchplay\fP 1 90  3 140  5 50
+.br
+will play track 1 with a pitch of 90%, track 3 with a pitch of 140%, and track 5 with a pitch of 50%.
+.PP
+.SH SEE ALSO
+cdda2wav(1)
+.PP 
+.SH "AUTHOR" 
+.PP 
+\fBcdda2was\fP was written by Joerg Schilling <js@cs\&.tu-berlin\&.de> and
+others.
+.PP 
+This manual page was written by Oleksandr Moskalenko
+<malex@tagancha\&.org>, for
+the Debian GNU/Linux system\&.  It may be used by other distributions
+without contacting the author\&.  Any mistakes or omissions in the
+manual page are my fault; inquiries about or corrections to this
+manual page should be directed to me (and not to the primary author)\&.
diff -urNad cdrtools-2.01.01~/doc/readmult.man cdrtools-2.01.01/doc/readmult.man
--- cdrtools-2.01.01~/doc/readmult.man	1970-01-01 01:00:00.000000000 +0100
+++ cdrtools-2.01.01/doc/readmult.man	2006-03-20 23:08:28.527365632 +0100
@@ -0,0 +1,29 @@
+.TH "READMULT" "1" "Tue Feb 15 12:34:06 MST 2005" "" "readmult"
+
+.PP
+.SH "NAME"
+readmult \- a multitrack wrapper for cdda2wav
+.PP
+.SH "SYNOPSIS"
+.PP
+\fBreadmult\fP <track a> <title a> <track b> <title b> ...
+.PP
+.SH "DESCRIPTION"
+.PP
+\fBreadmult\fP allows simultaneous extraction and naming of multiple tracks
+with cdda2wav
+.PP
+.SH SEE ALSO
+cdda2wav(1)
+.PP 
+.SH "AUTHOR" 
+.PP 
+\fBcdda2was\fP was written by Joerg Schilling <js@cs\&.tu-berlin\&.de> and
+others.
+.PP 
+This manual page was written by Oleksandr Moskalenko
+<malex@tagancha\&.org>, for
+the Debian GNU/Linux system\&.  It may be used by other distributions
+without contacting the author\&.  Any mistakes or omissions in the
+manual page are my fault; inquiries about or corrections to this
+manual page should be directed to me (and not to the primary author)\&.
